view/frontend/web/js/view/payment/method-renderer/multishipping/adyen-ideal-method.js

```diff
@@ -0,0 +1,182 @@
+/**
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2021 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+/*global define*/
+define([
+    'jquery',
+    'Adyen_Payment/js/view/payment/method-renderer/adyen-ideal-method',
+    'Adyen_Payment/js/model/adyen-payment-service',
+    'Adyen_Payment/js/model/adyen-configuration',
+    'Magento_Checkout/js/model/quote'
+], function (
+    $,
+    Component,
+    adyenPaymentService,
+    adyenConfiguration,
+    quote
+) {
+    'use strict';
+
+    return Component.extend({
+        defaults: {
+            template: 'Adyen_Payment/payment/multishipping/abstract-form'
+        },
+
+        selectPaymentMethod: function () {
+            $('#stateData').val(JSON.stringify(this.paymentComponent.data));
+            return this._super();
+        },
+
+        buildComponentConfiguration: function(paymentMethod, paymentMethodsExtraInfo) {
```

Comment:
`buildComponentConfiguration` shares a duplicate code for all multishipping payment method renderers. Is it possible to move it to a JS helper and call it from different method renderers?